exercises/practice/series/series.rakutest

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env raku
+use Test;
+use lib $?FILE.IO.dirname;
+use Series;
+
+cmp-ok( # begin: 7ae7a46a-d992-4c2a-9c15-a112d125ebad
+    series("1", 1),
+    "eqv",
+    ["1"],
```

Comment:
I'd personally prefer a list for the sake of consistency. I tend to choose lists over arrays if I have no intention of assigning/modifying the contents: https://github.com/exercism/raku/blob/main/exercises/practice/all-your-base/all-your-base.rakutest

Reply:
Agreed and completed.